<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputQueryForm</class>
 <widget class="QWidget" name="inputQueryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Значения для запроса</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер столбца   --&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setValuesForQueryPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Подтвердить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер строки    --&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rowLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
